micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ForgotOrganizationTooltip.js

```
@@ -0,0 +1,56 @@
+        };
+    }, []);
+    return (
+            <div ref={wrapperRef} className="loginSignUpSelector" style={{ position: "relative", marginTop: "-2rem" }}>
+                <Button
+                        label={t(`SB_FORGOTORGANIZATION_TOOLTIP`)}
+                        variation={"link"}
+                        size={"small"}
+                        onClick={() => setShowTip((prev) => !prev)}
+                        // isSuffix={true}
+                        style={{ marginBottom: "0.5rem", paddingLeft: "0.2rem" }}
+                ></Button>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)